src/main/java/com/github/alexthe666/iceandfire/compat/thaumcraft/ThaumcraftCompat.java

```diff
@@ -18,228 +22,266 @@
 public class ThaumcraftCompat {
 
     static final Aspect MYTHICAL = new Aspect("mythus", 0XD9D5AB, new Aspect[]{Aspect.BEAST, Aspect.MAGIC}, new ResourceLocation("iceandfire:textures/thaumcraft/mythical.png"), 1);
-    static final Aspect DRAGON = getOrCreateAspect("draco", 0XA2271F, new Aspect[]{ThaumcraftCompat.MYTHICAL, Aspect.DESIRE}, new ResourceLocation("iceandfire", "textures/thaumcraft/dragon.png"), 1);
+    static Aspect DRAGON = getOrCreateAspect("draco", 0XA2271F, new Aspect[]{ThaumcraftCompat.MYTHICAL, Aspect.DESIRE}, new ResourceLocation("iceandfire", "textures/thaumcraft/dragon.png"), 1);
+
+    private final Queue<AspectList> dracoItems = new ArrayDeque<>(68);
 
     static Aspect getOrCreateAspect(String tag, int color, Aspect[] components, ResourceLocation image, int blend) {
         Aspect a = Aspect.getAspect(tag);
         if (a != null) return a;
         return new Aspect(tag, color, components, image, blend);
     }
 
+    /**
+     * AspectRegistryEvent fires in TC's post-init (I&F load order does not matter) ->
+     * Then Thaumic Additions post-init replaces our Draco aspect from the TC registry.
+     * So we need to remap the reference and all our items to the new Draco aspect.
+     */
+    public void remapDraco() {
+        // Nothing overwrote our Draco aspect.
+        if (DRAGON.equals(Aspect.getAspect("draco"))) {
+            dracoItems.clear();
+            return;
+        }
+        IceAndFire.logger.info("Detected a mod overwrote I&F's Draco aspect: remapping " + dracoItems.size() + " entries.");
+        Aspect draco = Aspect.getAspect("draco");
```

Comment:
Could this get renamed to something like `realDraco` to further set apart the actual draco aspect versus I&F's? Also, this can get moved to above the `DRAGON.equals` so we don't have to fetch the aspect twice.